CHANGELOG.rst

```diff
@@ -18,6 +18,10 @@ Version 2.7 (unreleased)
   to set an end date for the most current version without adding a new
   version.
 
+  A (read-only) property ``currentsize`` to ``BatchFactTable`` and
+  ``_BaseBulkloadable`` to get the number of inserted rows awaiting
+  to be loaded into the database table. (GitHub issue #23)
+
```

Comment:
There seems to be an Added missing somewhere in this sentence.

Reply:
Yes, that was to be consistent with the item above for closecurrent  (and they are both under the Added heading). But we could write "added" for both if we prefer that.

Reply:
Writing as if the heading is the first word is fine, although we should probably do so consistently for the entire changelog. For example, at the moment we write added under **Version 2.2**.